crates/nu-command/src/commands/dataframe/drop_duplicates.rs

```diff
@@ -0,0 +1,66 @@
+use crate::prelude::*;
+use nu_engine::WholeStreamCommand;
+use nu_errors::ShellError;
+use nu_protocol::{dataframe::NuDataFrame, Signature, SyntaxShape, Value};
+
+use super::utils::{convert_columns, parse_polars_error};
+
+pub struct DataFrame;
+
+impl WholeStreamCommand for DataFrame {
+    fn name(&self) -> &str {
+        "pls drop_duplicates"
+    }
+
+    fn usage(&self) -> &str {
+        "Drops duplicate values in dataframe"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("pls drop_duplicates")
```

Comment:
Same (there are a few other places, but I'll just mention these and you can update the others)